File: Python/fibonacci.py
def fib(n):
    anteriores=[0,1]
    retorno=0
    for i in range(n):
        retorno=sum(anteriores)
        anteriores = [anteriores[1],retorno]
    return anteriores[1]

#exemplo de como utilizar   
print(fib(9))

File: C/fibonacci.c
long long int fib(int n){
    long long int a1=0,a2=1,r=0;
    int i;
    for(i=0;i<n;i++){
        r=a1+a2;
        a1=a2;
        a2=r;
    }
    return r;
}
